docs/content/users/install/phpstorm.md

```diff
@@ -20,7 +20,18 @@ If you’re using WSL2 and running PhpStorm on the Windows side, PhpStorm can’
 
 It’s easiest to use the DDEV Integration Plugin, which you can install from [its landing page](https://plugins.jetbrains.com/plugin/18813-ddev-integration) or by searching the in-app marketplace (*Preferences* → *Plugins* → *Marketplace*) for “DDEV”. The integration plugin handles nearly everything on this page automatically, and works on all platforms.
 
-Install and enable the plugin, then [set up `phpunit`](#enabling-phpunit) since it doesn’t yet handle that for you.
+Install and enable the plugin, then [set up `phpunit`](#enabling-phpunit) since it doesn't yet handle that for you.
+
+### Additional Step for Mutagen
+
+Please note that if you use mutagen (If you use Colima you will use mutagen by default. If you use Docker Desktop, you may opt-into using Mutagen), you will need an additional configuration task to set up your PHP Interpreter even if you are using this plugin.
+
+1. Open Preferences
+2. Navigate to PHP
+3. Edit the Path Mappings
+4. Add a path mapping that maps Local: /Users/<your_username>/<your_project_root>/ to Remote:/var/www/html
+
+PhpStorm mistakenly maps your project root to `/var/tmp/html`. Whenever you run a test / process in phpstorm that requires file permissions you'll likely run into a permission error. If you continue to run into file permission errors fix the offending folder by setting permissive permissions on it. `chmod 777 <directory>`
```

Comment:
Rebased, still waiting for your response on this @cosmicdreams. I don't believe this matters unless using a custom tool like phpunit does it?

Reply:
I disagree @rfay , when the path mapping is wrong (if the project root is set to `/var/tmp/html`) then any IDE operation that needs to create/edit something would fail.  The improper path mapping occurs when mutagen is enabled.  Therefore the documentation to users that they should be aware of the potential problem and how to fix it should be included. 

The contexts that information is needed is:
- Is running PhpStorm (where the configuration change needs to be made)
- when running mutagen (the situation in which it error occurs)

So I think it should go here.

Reply:
The plugin does the correct mapping in the part that it's responsible for.  This shows a fresh setup, mutagen enabled, plugin newly installed, .idea directory removed:

<img width="987" alt="image" src="https://user-images.githubusercontent.com/112444/226391426-870f702d-c0f2-4469-8dfc-4a154c95ce2a.png">

Reply:
While I see the same in `Servers`, 

![Screenshot 2023-03-20 at 10 52 33 AM](https://user-images.githubusercontent.com/852080/226396035-a612e888-5db2-46c3-b1bb-19ab9f1cda83.png)

my testing showed that it is the `Path mappings` setting that is associated with the PHP Interpreter that is at fault here.

![Screenshot 2023-03-20 at 10 52 48 AM](https://user-images.githubusercontent.com/852080/226396121-fb0e2d7f-c00a-4fbb-8227-2b94946ec102.png)

Reply:
Do you see the same in the PHP interpreter settings?

Reply:
Just rebuilt a new D10 site from the quickstart.  I found:

1. I could run tests via the command line (according to my documentation, that I've created a PR for)
2. I could not run tests via the test runner.  I received the following error:

```[docker-compose://[/Users/chrisweber/Sites/d10/.ddev/.ddev-docker-compose-full.yaml]:web/]:php8.1 /opt/.phpstorm_helpers/phpunit.php --no-configuration --filter Drupal\\Tests\\node\\Functional\\NodeAccessCacheabilityWithNodeGrants --test-suffix NodeAccessCacheabilityWithNodeGrants.php /var/tmp/html/web/core/modules/node/tests/src/Functional
Testing started at 11:34 AM ...

Process finished with exit code 1
PHPUnit settings are not configured.```

Note, the path the test runner believed it should use began with /var/tmp/html...

Reply:
Right now, I'm questioning if the additional documentation is still needed.  After updating DDEV, I'm having trouble getting the bug to recur.

Reply:
I think it always happens, but you need to delete the .idea directory to start from scratch, and the php interpreter config is global I think, so you'll have to delete your existing php interpreters.

Reply:
I have yet to have someone else (other than @ultimike ) report that they experienced this issue.  I'm having a colleague test to see if she can get the same error.

I have to report that I am not experiencing the error anymore.  I think we might be about to close this issue without needing this addition.